src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -332,5 +332,55 @@ InModuleScope Arcus.Scripting.LogicApps {
                 Assert-MockCalled Stop-AzLogicAppRun -Scope It -Times 0
             }
         }
+        Context "Resubmitting failed Logic Apps runs" {
+             It "Resubmitting failed runs from Logic App history should succeed" {
+                # Arrange
+                $resourceGroupName = "codit-arcus-scripting"
+                $logicAppName = "arc-dev-we-rcv-unknown-http"
+                $startTime = '2023-01-01 00:00:00'
+
+                Mock Get-AzLogicAppRunHistory -MockWith {
+                    return @{
+                        Name = "test"
+                        Status = "Failed"
+                        StartTime = "2023-01-01 01:00:00"
+                    }
+                }
+
+                Mock Invoke-WebRequest -MockWith {
+                   return $null
+                }
+
+                # Act
+                { Resubmit-FailedAzLogicAppRuns -ResourceGroupName $resourceGroupName -LogicAppName $logicAppName -StartTime $startTime } | 
```

Comment:
Also another test with the end-time present?

Reply:
Unit test added